<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcome</class>
 <widget class="QDialog" name="welcome">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Processor Version Control System - Welcome</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCommandLinkButton" name="lnkNewProject">
       <property name="text">
        <string>Create New Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="lnkOpenProject">
       <property name="text">
        <string>Open Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lstRecentProjects"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
